modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,27 +50,43 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
+be targeting. Create the following OperatorGroup in the namespace where you will
+create the CSV. In this example, the `default` namespace is used:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: memcached-operator-group
+  namespace: default
+  spec:
+    targetNamespaces:
+    - default
+----
+
+.. Apply the Operator's CSV manifest to the desired namespace in the cluster:
 +
 ----
 $ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
 $ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
 $ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
 ----
 
-.. After applying this manifest, nothing has happened yet, because the cluster does
-not meet the requirements specified in our manifest. Create the RBAC rules and
-`CustomResourceDefinition` for the Memcached type managed by the Operator:
+.. After applying this manifest, nothing has happened yet, because the cluster
+does not meet the requirements specified in our manifest. Create the role, role
```

Comment:
You should start with the action. I'd say "When you apply this manifest, the cluster does not immediately change because it does not yet meet the requirements that you specified." as more info after the previous step.

I'd start this step with "To allow the Operator to deploy, create the..."